static/index.js

```diff
@@ -149,19 +149,30 @@ window.addEventListener('load', e => {
   setupNavigationSection();
 });
 
-const setupBadgeSection = () => {
-  const badgeSection = document.querySelector('#badge_section');
-  const badgeAPI = window.Badge || window.ExperimentalBadge;
+function setBadge(...args) {
+  if (navigator.setExperimentalAppBadge)
+    navigator.setExperimentalAppBadge.apply(navigator, args);
+  else if (window.ExperimentalBadge)
+    ExperimentalBadge.set.apply(null, args);
+}
 
-  if (!badgeSection) {
-    return;
-  }
-  // Check that the API is available.
-  if (!badgeAPI) {
-    console.log("Badge API not detected :'(");
+function clearBadge() {
+  if (navigator.clearExperimentalAppBadge)
+    navigator.clearExperimentalAppBadge();
+  else if (window.ExperimentalBadge)
+    ExperimentalBadge.clear();
+}
+
+function badgeAPIAvailable() {
+  return !!(navigator.setExperimentalAppBadge || window.ExperimentalBadge);
+}
 
-    // If the Badging API is not available, hide the section.
-    badgeSection.remove();
+const setupBadgeSection = () => {
+  const logs = new LogSection(document.querySelector('#badge_section'));
+  if (!badgeAPIAvailable) {
+    logs.logMessage("navigator.setExperimentalAppBadge and ExperimentalBadge "
```

Comment:
Optional nit: backticks for multiline comments?

Reply:
Neh